06-building-a-program-start-to-finish/writing-tests/output/spec.js

```diff
@@ -0,0 +1,166 @@
+/* My Test Setups */
```

Comment:
Please use the proper naming convention for this test file. Meet your peers to inquire on what is preferred.